### PR TITLE
Update CHANGELOG and README in master branch

### PR DESCRIPTION
**Pull request by roycaihw**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Copy the CHANGELOG of v17.17.0b1 and v18.17.0a1 from the release branches. Also update README to include v18. 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

/cc @yliaog @scottilee

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +125,7 @@ between client-python versions.
 | 12.0 Alpha/Beta | Kubernetes main repo, 1.16 branch    | ✗                             |
 | 12.0            | Kubernetes main repo, 1.16 branch    | ✓                             |
 | 17.0 Alpha/Beta | Kubernetes main repo, 1.17 branch    | ✓                             |
+| 18.0 Alpha/Beta | Kubernetes main repo, 1.17 branch    | ✓                             |
```
Comment (yliaog) [inline README.md:129]: 1.18 branch?

File: README.md [hunk 2]
```diff
@@ -87,6 +87,7 @@ supported versions of Kubernetes clusters.
 - [client 11.y.z](https://pypi.org/project/kubernetes/11.0.0/): Kubernetes 1.14 or below (+-), Kubernetes 1.15 (✓), Kubernetes 1.16 or above (+-)
 - [client 12.y.z](https://pypi.org/project/kubernetes/12.0.1/): Kubernetes 1.15 or below (+-), Kubernetes 1.16 (✓), Kubernetes 1.17 or above (+-)
 - [client 17.y.z](https://pypi.org/project/kubernetes/17.14.0a1/): Kubernetes 1.16 or below (+-), Kubernetes 1.17 (✓), Kubernetes 1.18 or above (+-)
+- [client 18.y.z](https://pypi.org/project/kubernetes/17.14.0a1/): Kubernetes 1.17 or below (+-), Kubernetes 1.18 (✓), Kubernetes 1.19 or above (+-)
```
Comment (yliaog) [inline README.md:90]: https://pypi.org/project/kubernetes/17.14.0a1 needs to be updated to 18.y.z

File: README.md (hunk 1 above)
Comment (roycaihw) [reply README.md:129]: done

File: README.md (hunk 2 above)
Comment (roycaihw) [reply README.md]: done. Nice catch!

**Pull request merged**